### ISSUE TITLE
FR: less args upon run

### ISSUE THREAD

**Issue by LeandroMarceddu** [GijsvanDulmen/PlaterFlow#15]
Using binary ./frominput.js ./baseconfig.js will probably be confusing to less-pc-literate people.

Maybe implement frominput.js (in perhaps another name) automatically and make the user provide only one more arg?

We should divide up which config settings we want static and which ones we want dynamic

**Comment by GijsvanDulmen** [comment]
Ok. Proposal:
1)
`./platerflow` => uses `settings.js` (previous baseconfig) and `prints.js` (previous frominput.js).

2) 
`./platerflow ./prints-for-customer.js` => uses `settings.js` and `prints-for-customer.js`

3) 
`./platerflow ./prints-for-customer.js ./other-settings` => uses `other-settings.js` and `prints-for-customer.js`

**Comment by LeandroMarceddu** [comment]
Proposal:

4. `./platerflow --argument1=value1 --set=set2` -> uses settings.js, one giant settings file.
This would require maybe an external/extra binary/app for generating a settingsfile to make it more convenient for them

**Comment by GijsvanDulmen** [comment]
Yeah, the fact that we use a config file based approach makes it simple to save a bunch of configuration files for a few different things you want to re-platter/slice/upload a lot.

**Comment by LeandroMarceddu** [comment]
Perhaps the introduction of sets can be used to generate a per-set settings-override? E.G load value on start up, then load values in set again, overwriting the general config one?

**Comment by GijsvanDulmen** [comment]
This makes a bit more easy. More work to be done on this part.
https://github.com/platerflow/PlaterFlow/commit/01f3e27a5900396fd24fca40972c9e46e4613f3b
